website/docs/r/vpn_site.html.markdown

```diff
@@ -96,6 +98,22 @@ A `link` block supports the following:
 
 * `speed_in_mbps` - (Optional) The speed of the VPN device at the branch location in unit of mbps.
 
+---
+
+A `o365_policy` block supports the following:
+
+* `breakout_category` - (Optional) A `breakout_category` block as defined above.
+
+---
+
+A `breakout_category` block supports the following:
+
+* `allow_category_enabled` - (Optional) Is allow category controlled? Defaults to `false`.
+
+* `default_category_enabled` - (Optional) Is default category controlled? Defaults to `false`.
+
+* `optimize_category_enabled` - (Optional) Is optimize category controlled? Defaults to `false`.
+
```

Comment:
i think this needs to be reworked as its not very intuitive what is being set.

what does it mean by breakout category? what is allow vs default vs optimise? should it be o365_breakout_policy? or o365_breakout_catagory_policy

allow_controlled = true etc?

Reply:
@neil-yechenwei

Reply:
@katbyte , I think service team would expand the sub properties of `o365_policy` to introduce other o365 policies in the future. So suggest keep `o365_policy`. I updated `breakout_category` to `traffic_category`. See more details from https://docs.microsoft.com/en-us/microsoft-365/enterprise/microsoft-365-vpn-split-tunnel?view=o365-worldwide.